test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -60,11 +60,11 @@ Describe "Get-ChildItem" -Tags "CI" {
             $files.Count | Should be 1
             $files[0].Name | Should Be $item_F
         }
-        It "Should give .sys file if the fullpath is specified with hidden and force parameter" -Skip:(!$IsWindows){
-            $file = Get-ChildItem -path "$env:SystemDrive\\pagefile.sys" -Hidden
+        It "Should find the hidden file if specified with hidden switch" -Skip:(!$IsWindows){
```

Comment:
I'd love to see this written more directly and support on non-Windows. How about this?
```
    It "Should find the hidden file if specified with hidden switch" {
        $null = new-item $TESTDRIVE/.file.txt # automatically hidden on Linux
        if ( $IsWindows ) {
            attrib +h $TESTDRIVE/.file.txt
        }
        (Get-ChildItem $TESTDRIVE).count | should be 0
        (Get-ChildItem -hidden $TESTDRIVE).Count | should be 1
    }
```